src/components/drawer/Swipeable.tsx

```diff
@@ -303,8 +303,8 @@ export default class Swipeable extends Component<PropType, StateType> {
       }
     });
 
-    if (toValue === rowWidth * LEFT_TOGGLE_THRESHOLD && onToggleSwipeLeft) {
-      onToggleSwipeLeft({rowWidth, leftWidth});
+    if ((toValue === rowWidth * LEFT_TOGGLE_THRESHOLD || this.dragFired) && onToggleSwipeLeft) {
+      onToggleSwipeLeft({rowWidth, leftWidth, released: true});
```

Comment:
You're triggering `onToggleSwipeLeft` twice. In `_handleDrag` and in `_animateRow`
Not sure if that intentional, I doesn't break anything, but it might be a possible future bug unless I'm missing something

Reply:
Yes. I'm triggering `onToggleSwipeLeft` twice, one with release = false and once with release = true so I can first change the item's position and only on release fire the `onToggleSwipeLeft` callback and the haptic trigger.

Reply:
Got ya!